image_proc/src/crop_decimate.cpp

```diff
@@ -46,7 +46,13 @@ void debayer2x2toBGR(
   int R, int G1, int G2, int B)
 {
   typedef cv::Vec<T, 3> DstPixel;  // 8- or 16-bit BGR
-  dst.create(src.rows / 2, src.cols / 2, cv::traits::Type<DstPixel>::value);
+#if CV_VERSION_MAJOR >= 3 && CV_VERSION_MINOR >= 2
```

Comment:
This condition looks wrong. Version 4.0 / 4.1 would evaluate to `FALSE` and 4.2 would evaluate to `TRUE`.